### DISCUSSION TITLE
Can we add route aliases for articles?

### DISCUSSION THREAD

**Discussion by farhan27555** [vuejs/vitepress#3669] [Q&A]
I'm currently using VitePress for a new docs site and I'm curious to know if route aliases are supported. Specifically, I'd like to know if it's possible to have multiple URLs pointing to the same article, similar to route aliases.
This feature would be great for maintaining backward compatibility with my previous docs site. I appreciate any insights or information you can provide.

**Comment by Karlstens** [comment]
Have you had a look into route rewrites? And if so, if they didn't address your requirements, how did they differ from what you were expecting/requiring?

https://vitepress.dev/guide/routing#route-rewrites

**Comment by farhan27555** [comment]
@Karlstens  What I need is the ability to use the default routing behavior of Vitepress along with some aliases for certain routes. I've written some rewrites but it's not exactly what I was looking for.
For example, I have a route `/introduction` and for that, I have a rewrite `/articles/204273555-introduction`. This would've been fine if I wanted the page to be only accessed through the second route but that's not the case. I want the pages to be accessible through both the normal routes and through the rewrites. But it seems adding the rewrite doesn't allow us to use the default route any longer.

From sidebar:
`
items: [
          {
            text: "Introduction",
            link: "/articles/204273555-introduction/index.md"
          }
        ]
`

And rewrites:
`"introduction.md": "articles/204273555-introduction/index.md",`
